Lesson_26/task2.py

```diff
@@ -0,0 +1,116 @@
+"""Task 2 - Implement in (__contains__) and len (__len__) methods for HashTable"""
+
+
+class MyHashTable:
+    """Implementation of HastTable"""
+    def __init__(self, max_length=0):
```

Comment:
Usually `None` is used for the default value in such cases (`0` would also work, however)